.gitignore

```
@@ -1 +1,2 @@
-zig-cache
+zig-*
+.zig-*
```

Let's be more explicit:
```suggestion
.zig-cache
zig-cache
zig-out
```

---

Reasonable choice, I thought of it as a bit of future-proofing in case Zig grows another Zig directory, but it's trivial to add it to `.gitignore`s if that happens.